### PR TITLE
Update the help output for the occ files:scan command

### PR DESCRIPTION
**Pull request by settermjd**
## Description

Ensures that the descriptions for all commands in `occ help files:scan` are rendered consistently, and grammatically correct.

## Related Issue

#29580.

## Motivation and Context

The help output for `occ files:scan` renders inconsistently, both for capitalisation at the start of sentences, as well as for full-stops at the end of sentences. This decreases the readability of the command's output and detracts from a completely professional appearance.

## How Has This Been Tested?

Re-ran the test suite.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests have passed.

### Also Relates To 

https://github.com/owncloud/documentation/pull/3542

### CODE REVIEW

**Comment by settermjd** [comment]
In hindsight, I think I should have created this PR as a series of commits, and not one great big one.

File: core/Command/User/SyncBackend.php [hunk 1]
```diff
@@ -68,14 +68,24 @@ public function __construct(AccountMapper $accountMapper,
...
-				'The php class name - e.g. "OCA\User_LDAP\User_Proxy". Please wrap the class name into double quotes. You can use the option --list to list all known backend classes'
+				'The PHP class name - e.g., "OCA\User_LDAP\User_Proxy". Please wrap the class name in double quotes. You can use the option --list to list all known backend classes.'
 			)
-			->addOption('list', 'l', InputOption::VALUE_NONE, 'list all known backend classes')
-			->addOption('missing-account-action', 'm', InputOption::VALUE_REQUIRED, 'action to do if the account isn\'t connected to a backend any longer. Options are "disable" and "remove". Use quotes. Note that removing the account will also remove the stored data and files for that account');
+			->addOption(
+				'list',
+				'l',
+				InputOption::VALUE_NONE,
+				'list all known backend classes'
```
Comment (phil-davis) [inline core/Command/User/SyncBackend.php]: You missed a capital "L" and full-stop here?

**Comment by phil-davis** [comment]
Seems "a good thing" to me.

**Comment by mmattel** [comment]
👍

File: core/Command/User/ListUserGroups.php [hunk 2]
```diff
@@ -50,11 +50,11 @@ protected function configure() {
 
 		$this
 			->setName('user:list-groups')
-			->setDescription('list groups for a user')
+			->setDescription("List a user's groups")
```
Comment (mmattel) [inline core/Command/User/ListUserGroups.php]: Maybe: „Lists the groups a user belongs to“

File: core/Command/User/ListUsers.php [hunk 3]
```diff
@@ -44,11 +44,11 @@ protected function configure() {
 
 		$this
 			->setName('user:list')
-			->setDescription('list users')
+			->setDescription('List users.')
 			->addArgument(
 				'search-pattern',
 				InputArgument::OPTIONAL,
-				'Restrict the list to users whose User ID contains the string'
+				'Restrict the list to users whose user ID contains the supplied string.'
```
Comment (mmattel) [inline core/Command/User/ListUsers.php]: Maybe: ...contains the optional search pattern

**Comment by DeepDiver1975** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! This pull request is being rebased automatically. Please DO NOT push while rebase is in progress or your changes would be lost!

**Comment by ownclouders** [comment]
Automated rebase was successful!

**Comment by mmattel** [comment]
@settermjd mind to complete the PR so it can be finally reviewed and merged before 10.0.4 gets released ?
The open doc PR is depending on this one too...

**Comment by mmattel** [comment]
@settermjd any updates on the open comments ?

**Comment by mmattel** [comment]
@PVince81 is it normal hat drone takes 18h and is still not finished...?

**Comment by PVince81** [comment]
@mmattel I heard about some hardware issues on which Drone and Jenkins run... I hope we can sort this out soon.

**Comment by mmattel** [comment]
@PVince81 glory, test passed 😄. Mind to give your (hopefully positive) review so it can be merged?

**Review by PVince81 — APPROVED** [review]
Fantastic! Consistency FTW!

👍

**Comment by PVince81** [comment]
@settermjd please backport to stable10 for 10.0.5

**Comment by mmattel** [comment]
@settermjd I have not seen the backport (or missed it...), may I remind you and can you refrence from the backport to here respectively to #29580, thanks

**Comment by phil-davis** [comment]
Backport stable10 #29959 
(it was an easy cherry-pick with no conflicts)

**Comment by mmattel** [comment]
thx @phil-davis

**Pull request merged**